third-party/rspamd/README.md

```diff
@@ -24,13 +25,39 @@ guice.extension.task=org.apache.james.rspamd.module.RSpamDTaskExtensionModule
 </listener>
 ```
 
-- Declare the RSpamD mailet for custom mail processing. Mailet pipeline Eg:
-
-```
-<mailet match="All" class="org.apache.james.rspamd.RSpamDScanner"></mailet>
-<mailet match="IsMarkedAsSpam=org.apache.james.rspamd.status" class="WithStorageDirective">
-    <targetFolderName>Spam</targetFolderName>
-</mailet>
+- Declare the RSpamD mailet for custom mail processing. 
```

Comment:
```suggestion
- Declare the Rspamd mailet for custom mail processing.
```
Now that Tung started to fix the typo... :)

Reply:
I will remain the threads here as a reminder in case Tung PR got merged first.
TBH I prefer that got merged eventually.